tests/e2e/default-mobile/navigation/more-options-menu.wdio-spec.js

```diff
@@ -0,0 +1,107 @@
+const commonPage = require('@page-objects/default/common/common.wdio.page');
+const reportPage = require('@page-objects/default/reports/reports.wdio.page');
+const utils = require('@utils');
+const placeFactory = require('@factories/cht/contacts/place');
+const reportFactory = require('@factories/cht/reports/generic-report');
+const personFactory = require('@factories/cht/contacts/person');
+const userFactory = require('@factories/cht/users/users');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const sms = require('@utils/sms');
+
+describe('More Options Menu - Offline User', () => {
+  const places = placeFactory.generateHierarchy();
+  const health_center = places.get('health_center');
+  let xmlReportId;
+  let smsReportId;
+
+  const contact = personFactory.build({
+    phone: '+12068881234',
+    place: health_center._id,
+    parent: { _id: health_center._id, parent: health_center.parent },
+  });
+
+  const offlineUser = userFactory.build({
+    isOffline: true,
+    place: health_center._id,
+    contact: contact._id,
+  });
+
+  const patient = personFactory.build({
+    parent: health_center
+  });
+
+  const xmlReport = reportFactory
+    .report()
+    .build(
+      { form: 'home_visit', content_type: 'xml' },
+      { patient, submitter: contact }
+    );
+
+  const smsReport = reportFactory
+    .report()
+    .build(
+      { form: 'P', patient_id: patient._id, },
+      { patient, submitter: offlineUser.contact, fields: { lmp_date: 'Feb 3, 2022', patient_id: patient._id }, },
+    );
+
+  before(async () => {
+    await utils.saveDocs([ ...places.values(), contact, patient]);
```

Comment:
Adding missing space at the end of the array
```suggestion
    await utils.saveDocs([ ...places.values(), contact, patient ]);
```